### PR TITLE
New /enr endpoint to fetch ENR from bootnode.

### PR DESCRIPTION
**Pull request by protolambda**
- New endpoint that fetches the ENR from the bootnode
- Discv5 dependency update, fix for Go 1.15
- Gitignore
- Update rumor dependency, some types moved

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,8 +2,11 @@
 
 Minimal bootnode utility, with ENR options, geth discv5 logging, and persisted local node DB. 
 
+And a minimal HTTP endpoint, to serve the local ENR, in common base64 ENR representation, on the `/enr` route.
+
 Options:
 ```
+      --api-addr string        Address to bind HTTP API server to. API is disabled if empty. (default "0.0.0.0:80")
```
Comment (kvbik) [inline README.md]: default is `8000` looks like - further in the code

Comment (protolambda) [reply README.md]: Oh whoops, forgot to update that. Wasn't sure about the best default

Comment (protolambda) [reply README.md]: Fixed in 01f4bdc3ae898cfaeff00c161dd010f38e3a48a1

**Comment by protolambda** [comment]
Discv5 for Go 1.15 works now :tada:

**Pull request merged**